sci-physics/rivet/rivet-3.1.11-r1.ebuild

```diff
@@ -19,25 +16,20 @@ if [[ ${PV} == 9999 ]]; then
 	inherit git-r3
 	EGIT_REPO_URI="https://gitlab.com/hepcedar/rivet"
 else
-	SRC_URI="https://www.hepforge.org/archive/rivet/${MY_PF}.tar.gz -> ${P}.tar.gz"
-	S=${WORKDIR}/${MY_PF}
+	SRC_URI="https://www.hepforge.org/archive/rivet/${MY_P}.tar.gz -> ${P}.tar.gz"
+	S=${WORKDIR}/${MY_P}
 	KEYWORDS="~amd64"
 fi
 
 LICENSE="GPL-3+"
 SLOT="4/${PV}"
-IUSE="+hepmc3 hepmc2 +zlib +python"
-REQUIRED_USE="
-	^^ ( hepmc3 hepmc2 )
-	python? ( ${PYTHON_REQUIRED_USE} )
-"
+IUSE="+zlib +python"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
 
-RDEPEND="
+DEPEND="
 	>=sci-physics/fastjet-3.4.0[plugins]
 	>=sci-physics/fastjet-contrib-1.048
-	hepmc2? ( sci-physics/hepmc:2=[-cm(-),gev(+)] )
-	hepmc3? ( <sci-physics/hepmc-3.3.0:3=[-cm(-),gev(+)] )
-
+	<sci-physics/hepmc-3.3.0:3=[-cm(-),gev(+)]
```

Comment:
I guess the plan is to get rid of hepmc:2 altogether from `::gentoo`? In some niche cases one could use hepmc:2 to achieve analyses impossible with hepmc:3 in rivet. So if hepmc:2 stays keeping that option might be good.

Reply:
Just in case [madgraph](https://github.com/gentoo/sci/blob/8c0a01109ea0b1c7c0a5def061208c448a5163c9/sci-physics/madgraph5/madgraph5-3.5.8.ebuild#L40) only works with/supports hepmc:2 and friends in `::sci` https://github.com/search?q=repo%3Agentoo%2Fsci%20hepmc2&type=code. Maybe hepmc:2 should be moved to `::sci` then.